client/rechallenge/transformers.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```

Comment:
This should be imported into the entry point of the app which is `./client/index.js` 

This way it is available everywhere and we won't have to remember to import it everytime we want to use `includes()`